x-pack/plugins/lists/common/constants.mock.ts

```diff
@@ -32,9 +34,25 @@ export const NAMESPACE_TYPE = 'single';
 
 // Exception List specific
 export const ID = 'uuid_here';
+export const ITEM_ID = 'some-list-item-id';
 export const ENDPOINT_TYPE = 'endpoint';
-export const ENTRIES = [
-  { field: 'some.field', match: 'some value', match_any: undefined, operator: 'included' },
+export const FIELD = 'host.name';
+export const OPERATOR = 'included';
+export const ENTRY_VALUE = 'some host name';
+export const MATCH = 'match';
+export const MATCH_ANY = 'match_any';
+export const LIST = 'list';
+export const EXISTS = 'exists';
+export const NESTED = 'nested';
+export const ENTRIES: EntriesArray = [
```

Comment:
NOTE: Since I'm using unions, it was having a difficult time reading the correct type when being used for tests.